pom.xml

```diff
@@ -28,7 +28,7 @@
   <description>A Collection of Extensions for Syndesis</description>
 
   <properties>
-    <syndesis.version>2.0-SNAPSHOT</syndesis.version>
+    <syndesis.version>1.11.0.fuse-780011-redhat-00001</syndesis.version>
```

Comment:
I wonder if we should instead use `1.12-SNAPSHOT`. The idea of the extension master is to be aligned with actual 1.12.x of syndesis repo. Of course, it would only build locally if you also have the 1.12-SNAPSHOT installed. If a user want to use a stable version should therefore use a branch that would be pointing to the proper 1.x syndesis version. We should create a new extension branch 1.x as soon as the 1.x.0 of syndesis is delivered.

Reply:
I think it is better to wait for 1.11.0 release, which I expect to do it today. Then update this PR.

Reply:
1.11.0 will be okey for the extension branch 1.11.x (which we must create as well). The extension master branch should be pointing to the latest core development branch: it was `master` previously, and now should be `1.12-SNAPSHOT` (we will have to bump again when we start a new version).

Reply:
or we can align also extension with the new versioning we chose for core syndesis, removing `master` and create `1.12.x` that will have anyway pointing to `1.12-SNAPSHOT`.

Reply:
Updated the pom to 1.11.0 syndesis release.
I will prepare another PR to release syndesis-extension 1.11.0. I think leaving master branch for syndesis-extension and poiting to 1.12-SNAPSHOT is ok for the moment.